docs/google.rst

```diff
@@ -456,6 +460,7 @@ Example::
 
     google {
         project = 'your-project-id'
+        location = 'europe-west2'
         zone = 'europe-west1-b'
```

Comment:
Then, the `location` could be omitted from the example snippet. Do you agree?

Reply:
agreed. I'll remove the example snippets but leave the definition part and submit an updated pr

Reply:
thanks!